posthog/api/event.py

```diff
@@ -299,11 +300,17 @@ def session_events(self, request: request.Request, *args: Any, **kwargs: Any) ->
     # /event/session_recording
     # params:
     # - session_recording_id: (string) id of the session recording
+    # - save_view: (boolean) save view of the recording
```

Comment:
Nit: the name confused me at first. I think something like `record_watched` or something like that would make it more clear that you're storing that the user has already watched a given session recording.

Reply:
Problem with `record_viewed` is that it sounds like another filter - whereas this mutates state.